scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/SortedBucketIO.java

```diff
@@ -600,7 +600,7 @@ public abstract static class Write<K1, K2, V> extends PTransform<PCollection<V>,
 
     abstract int getSorterMemoryMb();
 
-    abstract FileOperations<V> getFileOperations();
+    public abstract FileOperations<V> getFileOperations();
```

Comment:
We could get around the signature change by adding a new method to SortedBucketIO like:

```java
public Iterator<T> tapBucketFile(ResourceId resourceId) {
   return getFileOperations().open(resourceId);
}
```

but this seems overtly hacky to me - plus, we re-create a FileOperations instance for every bucket/shard combo

Reply:
tbh it might be simplest to just make all these getters public 🤷‍♀️